invenio_app_ils/mail/tasks.py

```diff
@@ -45,6 +54,27 @@ def get_recipients(recipients):
     return recipients
 
 
+def create_email_log_entry(data, log_message):
+    """Create an entry in the email log db."""
+    patron = User.query.filter_by(email=data["recipients"][0]).one_or_none()
```

Comment:
unfortunately `patron` could be `None` (`one_or_none`)...
I would change this to
```
from sqlalchemy.orm.exc import NoResultFound
try:
    patron = User.query.filter_by(email=data["recipients"][0]).one()
except NoResultFound as ex:
    current_app.logger.exception(exception)
    return

```